scene/gui/spin_box.cpp

```diff
@@ -167,7 +169,7 @@ void SpinBox::gui_input(const Ref<InputEvent> &p_event) {
 		if (drag.enabled) {
 			drag.diff_y += mm->get_relative().y;
 			float diff_y = -0.01 * Math::pow(ABS(drag.diff_y), 1.8f) * SIGN(drag.diff_y);
-			set_value(CLAMP(drag.base_val + get_step() * diff_y, get_min(), get_max()));
+			set_value(CLAMP(drag.base_val + step * ceil(diff_y), get_min(), get_max()));
```

Comment:
What is the purpose of newly added `ceil()`?

Reply:
> What is the purpose of newly added `ceil()`?

I have no idea, it was in the original PR, I can remove it if needed

Reply:
Yeah, it doesn't seen necessary. It can be removed.